rdr_service/message_broker/message_broker.py

```diff
@@ -0,0 +1,105 @@
+from werkzeug.exceptions import BadRequest
+import httplib2
```

Comment:
Since we have `requests` in our venv, and its a wider used lib, can we utilize that for the `http` interactions?
https://python.libhunt.com/compare-requests-vs-httplib2